Formula/swift-format.rb

```diff
@@ -23,14 +23,19 @@ class SwiftFormat < Formula
   # out of the box on Xcode-only systems due to an incorrect sysroot.
   pour_bottle? only_if: :clt_installed
 
-  depends_on xcode: ["13.3", :build]
+  depends_on xcode: ["14.0", :build]
 
   uses_from_macos "swift"
 
   def install
-    system "swift", "build", "--disable-sandbox", "-c", "release"
+    system "swift", "build", "--disable-sandbox", "-c", "release", "-Xlinker", "-rpath", "-Xlinker", rpath
     bin.install ".build/release/swift-format"
     doc.install "Documentation/Configuration.md"
+
+    if OS.mac?
+      deuniversalize_machos ".build/release/lib_InternalSwiftSyntaxParser.dylib"
+      lib.install ".build/release/lib_InternalSwiftSyntaxParser.dylib"
```

Comment:
Also, not sure we want to put this in `lib` where other things could pick it up.

Reply:
Yes, definitely not.